sematic/api/endpoints/runs.py

```diff
@@ -219,6 +221,35 @@ def schedule_run_endpoint(user: Optional[User], run_id: str) -> flask.Response:
     return flask.jsonify(payload)
 
 
+@sematic_api.route("/api/v1/runs/<run_id>/logs", methods=["GET", "PUT"])
```

Comment:
PUT over POST because PUT is idempotent, as is this call. POST is not necessarily idempotent.